src/Docker/Runner/VariablesContext.php

```diff
@@ -17,20 +17,20 @@ public function __construct(array $configurationRow)
     }
 
     public function __isset($name)
-    {
-        return true;
-    }
-
-    public function __get($name)
     {
         if (isset($this->values[$name])) {
-            return $this->values[$name];
+            return true;
         } else {
             $this->missingVariables[] = $name;
-            return '{{ ' . $name . ' }}';
+            return false;
         }
     }
 
+    public function __get($name)
+    {
+        return $this->values[$name];
+    }
```

Comment:
Takto mi to pride lepsie.

Reply:
jo, ten mustas si vola pred get to isset, takze v getu uz je to vzdy set